config/drivetrain_config.yaml

```diff
@@ -1,10 +1,10 @@
-drivetrain_node:
+/**:
   ros__parameters:
-        HALF_WHEEL_BASE: 0.7127  # HALF of the wheelbase in meters # TODO: Verify this on the final robot
-        HALF_TRACK_WIDTH: 0.36585  # HALF of the trackwidth in meters # TODO: Verify this on the final robot
-        STEERING_MOTOR_GEAR_RATIO: 20 # TODO: Ask mechanical what this value will be on the real robot
-        FRONT_LEFT_MAGNET_OFFSET: 0 # TODO: Calibrate this on the final robot
-        FRONT_RIGHT_MAGNET_OFFSET: 0 # TODO: Calibrate this on the final robot
-        BACK_LEFT_MAGNET_OFFSET: 0 # TODO: Calibrate this on the final robot
-        BACK_RIGHT_MAGNET_OFFSET: 0 # TODO: Calibrate this on the final robot
-        GAZEBO_SIMULATION: False # Set to true if running in Gazebo
+    HALF_WHEEL_BASE: 0.7127 # HALF of the wheelbase in meters # TODO: Verify this on the final robot
+    HALF_TRACK_WIDTH: 0.36585 # HALF of the trackwidth in meters # TODO: Verify this on the final robot

```

Comment:
Verify/remove comment

Reply:
We should measure this to get the exact values

Reply:
done